reframe/core/modules.py

```diff
@@ -939,3 +941,111 @@ def emit_load_instr(self, module):
 
     def emit_unload_instr(self, module):
         return ''
+
+
+class SpackImpl(ModulesSystemImpl):
+    '''Base class for TMod Module system (Tcl).'''
+
+    def __init__(self):
+        # Try to figure out if we are indeed using the TCL version
+        try:
+            completed = osext.run_command('spack -V')
+        except OSError as e:
+            raise ConfigError(
+                'could not find a sane Spack installation') from e
+
+        self._version = completed.stdout.strip()
+
+        # self._version = version
+        # try:
+        #     # Try the Python bindings now
+        #     completed = osext.run_command(self.modulecmd())
+        # except OSError as e:
+        #     raise ConfigError(
+        #         'could not get the Python bindings for TMod: ' % e) from e
+
+        # if re.search(r'Unknown shell type', completed.stderr):
+        #     raise ConfigError(
+        #         'Python is not supported by this TMod installation')
```

Comment:
@victorusu If there are no Python bindings for `spack load`, then the only thing we can do for this backend is to emit the right commands and list the loaded modules.

Reply:
yes

Reply:
@rsarm You need to remove these commented lines.